### DISCUSSION TITLE
[03/10/2024] Score Community Meeting

### DISCUSSION THREAD

**Discussion by sujaya-sys** [score-spec/spec#106] [Community meetings]
**Info**

| Community call | Info |
|:-----------|:------------|
| Meeting Link | Join via [Google Meet](https://meet.google.com/znt-usdc-hzs) or call +49 40 8081618260 (Pin: 599 887 196)
| Meeting Agenda & Notes | Add to our agenda or review minutes [here](https://github.com/score-spec/spec/discussions/categories/community-meetings)
| Meeting Time | 1:00-2:00pm UTC, every first Thursday of the month

**Agenda**

* Review of [open PRs](https://github.com/pulls?q=is%3Aopen+is%3Apr+archived%3Afalse+user%3Ascore-spec) and  newly added [issues](https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+user%3Ascore-spec+)

* Next steps score-azure implementation

**Comment by mathieu-benoit** [comment]
Suggestions:
- New Community page in the docs: https://docs.score.dev/docs/community/
  - Feedback?
  - How to link it from https://score.dev/community/?
- https://docs.google.com/presentation/d/1ANil2uJC6-gzdb6geiB750PTUsAvMEkkNhgRdMYfpaA
  - https://clotributor.dev/search?ts_query_web=Score
  - https://github.com/cncf/tag-contributor-strategy/pull/721 --> `Makefile` from here: https://landscape.cncf.io/api/projects/all.json and here https://github.com/cncf/landscape/blob/master/landscape.yml#L7012
  - https://github.com/cncf/clomonitor/pull/1597
- Score at Cloud Native Rejekts NA 2024: https://cfp.cloud-native.rejekts.io/cloud-native-rejekts-na-salt-lake-city-2024/talk/WHR8SY/ 
- Ideas for blog posts:
  - What has changed/What's new since https://score.dev/blog/score-joins-the-cncf-as-a-sandbox-project/ and https://score.dev/blog/a-new-chapter-for-score-introducing-our-reference-implementations-for-compose-k8s/.
  - Score at KubeCon NA 2024 like we did: https://score.dev/blog/score-at-kubecon-eu-in-paris/
  - Something more simple but same idea as https://medium.com/@mabenoit/what-if-the-azure-samples-aks-store-demo-was-using-score-655c55f1c3dd
  - Where Score fits in WG landscape by Susa.

**Comment by mathieu-benoit** [comment]
Some notes from the meeting:
- Attendees: @sujaya-sys @delca85 @chrishumanitec @mathieu-benoit 
- `score-azure` implementation in progress. Do we track this somewhere, @sujaya-sys?
- 1 interest to start https://github.com/score-spec/score-compose/issues/178, great!
- Based on the new Community doc https://docs.score.dev/docs/community/, let's update the Community page too: https://github.com/score-spec/score.dev/issues/14
- Susa and Chris discussed that a new issue should be filed to document and change the spec to support an array with files. @chrishumanitec and @sujaya-sys, is that correct?
- We rediscussed the importance of https://github.com/score-spec/score-k8s/issues/25. Should Score have an alternative between either "create your own implementation" or "just patch the generated manifests with `yq`"? We don't know yet, but we will need to file a new issue in the spec to open a discussion and first capture the examples we got so far. And let's see from there. There is also tackling this concept: https://github.com/score-spec/score-k8s/issues/35.
- For blog posts, Mathieu and Susa will work on them. For the KubeCon NA one, Mathieu to confirm with Florian.
- On the [Score page](https://landscape.cncf.io/?_gl=1*yf2i6i*_gcl_au*MTExNDA5NTkxOS4xNzI3MDk1Njk5*_ga*MTgwNDU0MTUwNi4xNjgxMjI2NDgw*_ga_VWZ4V8CGRF*MTcyNzk2MjMyNC42OC4xLjE3Mjc5NjIzMjcuMC4wLjA.&item=app-definition-and-development--application-definition-image-build--score), the "language to contribute" is `Makefile` because the main repo is the spec, and not the impementations in Go for example. It could be misleading for the other repos like `score-compose` and `score-helm`, but that's how it is. Score is first a spec and then have implementations. To mitigate this, https://clotributor.dev/search?ts_query_web=Score is where people can see/search how they can contribute to Score, so it's important to use the `good-first-issue` label in all issues and all repos when appropriate.
